Userland/Utilities/comm.cpp

```diff
@@ -93,35 +103,41 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto cmp = [&](String const& str1, String const& str2) {
         if (case_insensitive)
             return strcasecmp(str1.characters(), str2.characters());
-        else
-            return strcmp(str1.characters(), str2.characters());
-    };
-
-    auto process_remaining = [](String const& fmt, auto& file, int& count, bool print) {
-        while (file->can_read_line()) {
-            ++count;
-            auto line = file->read_line();
-            if (print)
-                outln(fmt, line);
-        }
+        return strcmp(str1.characters(), str2.characters());
```

Comment:
`str1 == str2`?

Reply:
`strcmp` returns a number that tells you whether str1 goes before or after str2 alphabetically, which is used on lines 140, 145 and 151.